src/foam/graphics/GreekView.js

```diff
@@ -0,0 +1,189 @@
+foam.CLASS({
+  package: 'foam.graphics',
+  name: 'GreekView',
```

Comment:
Let's rename this LegendView or MapView or ZoomMapView or PanView or something else. I can't find anywhere online that this is referred to as a GreekView. I have an encyclopedia of visual components at home that I'll check.

Reply:
Went with ZoomMapView